### DISCUSSION TITLE
A gap on inverse pole figure

### DISCUSSION THREAD

**Discussion by kevindurantnets** [mtex-toolbox/mtex#2235] [Ask Anything] [answered]
Hello all, 
I’m trying to plot inverse pole figures of a FCC high entropy alloy, but there is a gap at [-111] point, which is circled by a red dot. My code is shown below. How could I solve this problem?

CS = {... 
  'notIndexed',...
  crystalSymmetry('m3m', [3.6 3.6 3.6], 'mineral', 'Nickel', 'color', [0.53 0.81 0.98])};

plotIPDF(ebsd.orientations,[vector3d.X,vector3d.Y,vector3d.Z])

![inverse pole figure](https://github.com/user-attachments/assets/427f3fff-a2a8-4c89-bb41-350ae5a720b0)

Thanks in advance.

**Comment by zmichels** [comment]
Huh... seems like a plot label is maybe sorta overlaying the corner? You could try turning off the notations. What about trying (just to see) if a different plotting size/preference makes a difference...

like if you add inside the plot command... 

..., 'figSize', 'tiny')

or 'huge'... 

just specifying and/or changing it to see if/how it affects the corner point.

**Comment by kevindurantnets** [comment]
If I shink the figure, the gap will be small, but it still exist. This problem does not seem to exist in titanium inverse pole figure.

![inverse pole figure2](https://github.com/user-attachments/assets/28bb9186-5951-44de-9cff-82a4cb73e4a0)

**Comment by ralfHielscher** [comment, marked as answer]
This some kind of a bug. The reason is that the vertices of the fundamental sector are not hard coded in MTEX. You can overcome this problem by editing the file `mtex/geometry/@sphericalRegion/plot.m` at line 27 and increase the value `2161` to something bigger, e.g. `8641`
```matlab
% plot the region
omega = linspace(0,2*pi,8641);
```
Ralf.

**Comment by kevindurantnets** [comment]
Dear sir, 
The problem has been solved. Thanks for your help.
